### DISCUSSION TITLE
Support for Sprite Extruder Pro

### DISCUSSION THREAD

**Discussion by stlprettyboy** [mriscoc/Ender3V2S1#273] [Q&A] [answered]
I change my s1 stock extruder to the s1 pro so i can i get it to go over the stock 260'

**Comment by mriscoc** [comment, marked as answer]
Use the T13 version from: https://github.com/mriscoc/Special_Configurations/releases/tag/AM3950T13

**Comment by stlprettyboy** [comment]
But I have a S1 with a cr touch So does it matter

**Comment by jufesa** [comment]
hi, excuse me, i´m new, how to use the command c104??

**Reply by mriscoc** [reply]
Check the video section in the Wiki.
